_overviews/scala3-book/scala-for-python-devs.md

```diff
@@ -89,22 +88,23 @@ Also at a programming level, these are some of the differences you’ll see ever
 - Scala is statically typed, so you declare types for method parameters, method return values, and in other places
 - Pattern matching and `match` expressions are used extensively in Scala  (and will change the way you write code)
 - Traits are used heavily in Scala; interfaces and abstract classes are used less often in Python
-- Scala’s *contextual abstractions* and *term inference* provide a collection of different features:
-  - *Extension methods* let you easily add new functionality to classes using a clear syntax
-  - *Multiversal equality* lets you limit equality comparisons---at compile time---to only those comparisons that make sense
-- Scala has state-of-the-art open source functional programming libraries
+- Scala’s [contextual abstractions][contextual] and _term inference_ provide a collection of different features:
+  - [Extension methods][extension-methods] let you easily add new functionality to classes using a clear syntax
+  - [Multiversal equality][multiversal] lets you limit equality comparisons---at compile time---to only those comparisons that make sense
+- Scala has state-of-the-art open source functional programming libraries (see the [“Awesome Scala” list](https://github.com/lauris/awesome-scala))
 - You can create your own “control structures” and DSLs, thanks to features like objects, by-name parameters, infix notation, optional parentheses, extension methods, higher-order functions, and more
-- Scala code can run in the JVM and even be compiled to native images (using Scala Native and GraalVM) for high performance
-- Many other goodies: case classes, companion classes and objects, macros, union and intersection types, toplevel definitions, numeric literals, multiple parameter lists, and more
-
+- Scala code can run in the JVM and even be compiled to native images (using [Scala Native](https://github.com/scala-native/scala-native) and [GraalVM](https://www.graalvm.org)) for high performance
+- Many other goodies: case classes, companion classes and objects, macros, [union][union-types] and [intersection][intersection-types] types, [toplevel definitions][toplevel], numeric literals, multiple parameter lists, and more
 
 
 ### Features compared with examples
 
 Given that introduction, the following sections provide side-by-side comparisons of Python and Scala programming language features.
 
-> The general Python standard is to indent code with four spaces, but in the following examples only two spaces are used.
-> This is only done so the examples can be shown side by side.
+{% comment %}
+TODO: Update the Python examples to use four spaces. I started to do this, but then thought it would be better to do that in a separate PR.
+{% endcomment %}
+
```

Comment:
Maybe we can record TODOs in issues, rather than inline comments. That would make it easier to track them and would not show up in diffs.

Reply:
Sounds good. I’ll work on removing them all this week.